src/main/java/com/force/api/ApiVersion.java

```diff
@@ -29,4 +29,16 @@
 	
 	public String toString() { return v; }
 
+    public static ApiVersion fromValue(String value) {
```

Comment:
@cswendrowski Can I delete this method? Or do you still need it? I'd prefer to not add anything to ApiVersion at this point.